Header header
geometry_msgs/PoseWithCovariance pose
geometry_msgs/TwistWithCovariance twist
string[] activity_labels
float64[] activity_confidences
string activity
float64 activity_confidence
string[] class_labels
float64[] class_confidences
string class_label
float64 class_confidence